Library/Homebrew/os/mac/xcode.rb

```diff
@@ -74,7 +74,7 @@ def uncached_version
         # This is a separate function as you can't cache the value out of a block
         # if return is used in the middle, which we do many times in here.
 
-        return "0" unless OS.mac?
+        return "0" unless OS.mac? && MacOS.can_build?
```

Comment:
Do you mind using `!MacOS::Xcode.installed? || !MacOS::CLT.installed?` instead? `MacOS.can_build?` is pretty unclear in its purpose here.